src/Pact/Types/SigData.hs

```diff
@@ -79,6 +79,7 @@ data SigData a = SigData
   , _sigDataCmd :: !(Maybe a)
   } deriving (Eq,Show,Generic)
 
+-- TODO: This is wrong: It will only parse ED25519 signatures.
```

Comment:
todo?

Reply:
I propose we delete this TODO and leave the `SigData` parser as it is. `SigData` is used by the pact CLI for updating commands (adding new sigs) after-the-fact. The CLI tool is not going to a WebAuthn client, so we will not need to parse WebAuthn signatures.